README.md

```diff
@@ -102,8 +102,25 @@ image classification, just run the following to start the pipeline:
 $ pipcook run examples/pipelines/mnist-image-classification.json
 ```
 
-__NOTICE__: the last two examples are using Boa (pipcook python bridge layer). 
-Before run them, you need to setup Python environment. See [here](docs/tutorials/want-to-use-python.md) for more information
+__NOTICE__: The last two examples are using Boa (pipcook python bridge layer). Boa is not responsible for installing specific python packages currently. You can use `pipcook bip` as an alias for `pip` and install packages. To make pipeline-object-detection working, please make sure you have dependencies specified in [detectron2 installation reference][]. See [here](docs/tutorials/want-to-use-python.md) for more information about Boa.
+
+### Serving
```

Comment:
The "Serving" and "Playground" in the same level, how about changing the "Playground" to "Pipeline"? And separate the `pipboard` to another section before the "Pipeline"?